modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/PersistenceTest.cs

```diff
@@ -224,68 +227,74 @@ public void TestGridActivationNoPersistence()
         [Test]
         public void TestBaselineTopology()
         {
-            Environment.SetEnvironmentVariable("IGNITE_BASELINE_AUTO_ADJUST_ENABLED", "false");
+            var oldAutoAdjustEnabled = Environment.GetEnvironmentVariable(BaselineAutoAdjustEnabledVar);
```

Comment:
Just a suggestion. I noticed at least one more usage with the same case - TestIgniteHome. 
Probably it's a good point for refactoring.

Extract a single IDIsposable class,
setup new var in ctor
resolve old in dispose

Thoughts?

Reply:
Agree, done